### PR TITLE
fix(sheets-ui): can't edit after dispose

### PR DESCRIPTION
**Pull request by weird94**
close #xxx

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Review by wzhudev — CHANGES_REQUESTED** [review]
这种比较 hack 的修复需要写注释保留上下文，即为什么延时 100ms 能解决问题，不然后面不好维护

File: packages/sheets-ui/src/controllers/render-controllers/editor-bridge.render-controller.ts [hunk 1]
```diff
@@ -74,7 +74,9 @@ export class EditorBridgeRenderController extends RxDisposable implements IRende
         this._initSelectionChangeListener(d);
         this._initEventListener(d);
         this._commandExecutedListener(d);
-        this._initialKeyboardListener(d);
+        setTimeout(() => {
```
Comment (Jocs) [inline packages/sheets-ui/src/controllers/render-controllers/editor-bridge.render-controller.ts:77]: Why use settimeout？

Comment (weird94) [reply packages/sheets-ui/src/controllers/render-controllers/editor-bridge.render-controller.ts:77]: 现象是这个sheet renderUnit创建的时候 cell-edtior的doc还没有创建 导致事件监听没有加上

Comment (Jocs) [reply packages/sheets-ui/src/controllers/render-controllers/editor-bridge.render-controller.ts:77]: 100ms 就能保证doc创建了吗？doc cell editor 是在什么时候创建？

**Comment by weird94** [comment]
> 这种比较 hack 的修复需要写注释保留上下文，即为什么延时 100ms 能解决问题，不然后面不好维护

改了一下实现

**Review by Jocs — APPROVED** [review]

**Pull request merged**